templates/glitch.html

```diff
@@ -45,6 +45,7 @@
         </div>
         <div class="web-codelab-instructions__heading-title">
           <h1>{{title}}</h1>
+          <a href="https://github.com/GoogleChrome/web.dev/tree/master/content{{relatedGuideHref}}/codelab-{{glitch}}.md" target="_blank" rel="noopener">Edit this codelab on GitHub</a>
```

Comment:
Nit: maybe use "Edit this page" for everything. More concise and universal across pages.